contents/handbook/engineering/how-we-review.md

```diff
@@ -0,0 +1,42 @@
+---
+title: How we review PRs
+sidebar: Handbook
+showTitle: true
+---
+
+Almost all PRs made to repositories on PostHog will need a review from another engineer. We do this because almost every time we review a PR we find a bug, a performance issue, unnecessary code or UX that could have been confusing.
+
+## How to review
+
+1. Have a flick through the code.
```

Comment:
Suggestion:

```
1. Look through the description, screenshots and  linked issues to understand the goal and user change of the PR.
```

This should act as a forcing function to write good descriptions/issues as well (Marius is great at this) that aid with the review.